src/main/scala/explicitdeps/ExplicitDepsPlugin.scala

```diff
@@ -38,7 +38,8 @@ object ExplicitDepsPlugin extends AutoPlugin {
   lazy val undeclaredCompileDependenciesTask = Def.task {
     val log = streams.value.log
     val projectName = name.value
-    val allLibraryDeps = getAllLibraryDeps(compile.in(Compile).value.asInstanceOf[Analysis], log)
+    val csrCacheDirectoryValue = csrCacheDirectory.value.toPath.toString
```

Comment:
csrCacheDirectory is the Coursier cache, right? So I guess this key is only defined in sbt 1.3.0 or newer (or older sbt if you happen to be using the coursier plugin)?

I wonder if there's a nice way to say "get me the value of this key *if it exists*"...

Or maybe enough time has passed for us to stop supporting sbt 1.2.x or older. We can certainly stop supporting 0.13.x, that's for sure.

Reply:
Yes this key is defined only in sbt 1.3.0 and above